app/test_main.py

```diff
@@ -1,3 +1,49 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age,dog_age,expected", [
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
```

Comment:
The expected result for (23, 23) is [1, 1], which might not align with the logic if the second year is supposed to add additional human years. Verify the logic in `convert_to_human` to ensure consistency.